model/task/task.go

```diff
@@ -2393,31 +2390,66 @@ func (t *Task) IsPartOfSingleHostTaskGroup() bool {
 }
 
 func (t *Task) IsPartOfDisplay() bool {
-	dt, err := t.GetDisplayTask()
-	if err != nil {
-		grip.Error(message.WrapError(err, message.Fields{
-			"message":        "unable to get display task",
-			"execution_task": t.Id,
-		}))
-		return false
+	// if display task ID is nil, we need to check manually if we have an execution task
+	if t.DisplayTaskId == nil {
```

Comment:
what if it's not nil, but it's ""? Do we not need to check in that case?

Reply:
If it's explicitly set as a pointer to the empty string, then we know that there's no display task for this task, so we don't have to look it up from the DB (this is handled at line 2406)